### ISSUE TITLE
LibreOffice compatibility (ask to repair the file)

### ISSUE THREAD

**Issue by leiblix** [rzymek/opczip#10]
Recently, LibreOffice started reporting files generated by OpcZip as corrupted.

Fastexcel: https://github.com/dhatim/fastexcel/issues/486
Apache POI: https://github.com/apache/poi/pull/154

Bug report on LibreOffice is here https://bugs.documentfoundation.org/show_bug.cgi?id=163384#c5

The problem is kind of related https://github.com/rzymek/opczip/issues/9

Sizes are written as Long (ZIP64), but there is no zip64 extension provided => LibreOffice read just first 32 bites of these and the remaining bytes are detected like loopholes in ZIP.

**Comment by leiblix** [comment]
According to the latest message, they have read your article (https://rzymek.github.io/post/excel-zip64/) and will fix it in LibreOffice.

So basically, the package is not a fully valid zip64, but it is an industry standard defined by Excel. :-|

If they fix it (I hope so), I will close this task myself.

**Comment by rzymek** [comment]
Yes, exactly - this package was create to support large xlsx creation that is compatible with MS Excel. While navigating the not-so-well-defined Zip64 standard

**Comment by luccahuguet** [comment]
> Yes, exactly - this package was create to support large xlsx creation that is compatible with MS Excel. While navigating the not-so-well-defined Zip64 standard

I see. Did you notice any movement from libreOffice? thanks for the information
